src/scheduler/batch/tasks/populateMissingAnnualReviewDates/index.ts

```diff
@@ -22,8 +22,11 @@ export async function main() {
   updates
     .filter((result) => result.status !== "fulfilled")
     .forEach((failedResult) => {
-      // @ts-ignore
-      logger.error(failedResult.reason);
+      logger.error(
```

Comment:
`const logger = schedulerLogger.child({ method: "populateAnnualReviewDates" });` 

should produce logs that contain {method: "populateAnnualReviewDates"} at the end, so there's already context there (doesn't need to be re-added)